### PR TITLE
feat(standups): Responses grid with static prompt

### LINKED ISSUE #6201: Standup UI - Responses grid
Design - https://www.figma.com/file/bhiEOV4HK9Kv91HFsJmevV/Stand-up-Concepts?node-id=1138%3A72617
Design issue - https://github.com/ParabolInc/parabol/issues/5965 

Questions
 - How does it scale? is it the same as meetings view?

AC
- [ ] Grid with placeholders components is added

### PR DESCRIPTION
**Pull request by BartoszJarocki**
Fixes #6201 

For now, we'll use the same grid as we use on the meeting dashboard. In next iterations we'll improve and implement #6389.
Earlier discussion for the reference: https://github.com/ParabolInc/parabol/issues/5965

This PR also adds https://github.com/ParabolInc/parabol/issues/6200 as it was a little change.

### CODE REVIEW

**Comment by BartoszJarocki** [comment]
@jmtaber129 would you mind taking a look? thanks!

File: packages/client/components/TeamPromptMeeting.tsx
Comment (jmtaber129) [inline packages/client/components/TeamPromptMeeting.tsx:148]: +1 Would it make sense to use `child.key` here?  That would guarantee consistency between the `key` used by `useTransition` and the `key` used by React.

File: packages/client/components/TeamPromptMeeting.tsx
Comment (jmtaber129) [inline packages/client/components/TeamPromptMeeting.tsx:93]: +1 `id` and `phaseType` don't seem to be used anywhere

**Review by jmtaber129 — APPROVED** [review]

File: packages/client/components/TeamPromptMeeting.tsx
Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:75]: 🤦

File: packages/client/components/TeamPromptMeeting.tsx
Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:114]: yeah, I'm pretty sure it'll need to be adjusted to the card component anyway so I didn't want to guess. if you don't mind, I'll leave it as is

**Comment by BartoszJarocki** [comment]
@jmtaber129 I fixed every comment except for the mapping which I think should be aligned in the PR related to cards. thank you for the review, moving on to the maintainer review 🙏

**Comment by BartoszJarocki** [comment]
@mattkrick I'd be great if you could take a look 🙏 thank you!

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [inline packages/client/components/TeamPromptMeeting.tsx:91]: -1 fragment on `TeamPromptResponsesPhase` instead. it's a little more expressive way to say "we only care about this 1 phase in particular"

as it's written, it's saying, "look at EVERY phase & for EVERY phase, give me the TeamPromptResponsesStages"

Also, if you fragment on the phase, then the teamMember object is guaranteed non null, which is nice.

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [inline packages/client/components/TeamPromptMeeting.tsx:92]: -1 Could you please rename `TeamPromptResponsesPhase` to `TeamPromptResponsePhase`? (notice  the missing "s"). not a problem in this PR, but i didn't discover it until now

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [inline packages/client/components/TeamPromptMeeting.tsx:122]: -1 with the changes above, you can simplify this to:

```ts
 const responsePhase = meeting.phases.find((phase) => phase.__typename === 'TeamPromptResponsePhase')!
 const teamMembers responsePhase.stages.map((stage) => {...stage.teamMember, key: stage.teamMember.id}))
```

**Review by mattkrick — CHANGES_REQUESTED** [review]
some very minor client-side stuff, after that feel free to merge!

File: packages/client/components/TeamPromptMeeting.tsx
Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:122]: typescript is a little dumb in that case and `responsePhase` can be another type as well.
see the screenshot below
Screenshot 2022-04-19 at 15 35 16

do we already have some generic utility to generate type predicates? we'd need something like this and I just want to make sure I haven't missed anything in the codebase 
https://stackoverflow.com/questions/59050071/narrow-return-type-of-find-from-discriminated-union-array

File: packages/client/components/TeamPromptMeeting.tsx
Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:92]: no strong opinions here, the origin of the name is the fact that this phase is for all responses, not one. let me know if that makes sense. if you think it's still better to use `TeamPromptResponsePhase` I will fix

File: packages/client/components/TeamPromptMeeting.tsx
Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:91]: oh cool, will fix

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [reply packages/client/components/TeamPromptMeeting.tsx:92]: ah gotcha, that works!

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [reply packages/client/components/TeamPromptMeeting.tsx:122]: ah good call, we have `GQLType`, which does the trick. I'll write a little commit to showcase it

File: packages/client/components/TeamPromptMeeting.tsx
Comment (mattkrick) [inline packages/client/components/TeamPromptMeeting.tsx:109]: @BartoszJarocki let me know if this gets you whatcha need. if so, go ahead & merge this PR & we'll start using this more in the future

Comment (BartoszJarocki) [reply packages/client/components/TeamPromptMeeting.tsx:109]: yup, looks good! thank you!

**Pull request merged**
